src/mod_extdisco.erl

```diff
@@ -39,6 +40,27 @@ stop(Host) ->
     mod_disco:unregister_feature(Host, ?NS_EXTDISCO),
     gen_iq_handler:remove_iq_handler(ejabberd_sm, Host, ?NS_EXTDISCO).
 
+-spec config_spec() -> mongoose_config_spec:config_section().
+config_spec() ->
+    #section{
+        items = #{<<"service">> => #list{items = service(),
+                                         format = none}}
+        }.
+
+service() ->
```

Comment:
I'd name this function `service_spec` or `service_config_spec` to be more descriptive.